### PR TITLE
[EDU-2186] BUILD MENU PT main settings

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Translated main settings

### Additional links

[EDU-2186]: https://aziontech.atlassian.net/browse/EDU-2186?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 1]
```diff
@@ -0,0 +1,193 @@
+---
+title: Definir configurações principais
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:2]:
```suggestion
title: Como definir as configurações principais de uma edge application
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+menu_namespace: buildMenu
+---
+
+Quando você cria um edge application com a Azion, você tem uma série de configurações que pode modificar. Neste guia, você aprenderá a modificar os protocolos, portas de entrega e o tipo de criptografia que sua aplicação usa para se comunicar com seus usuários finais.
+
+:::note
+Não tem uma edge application? [Comece com um template](/en/documentation/products/start-with-a-template/).
+:::
+
+## Via RTM
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:16]:
```suggestion

---

## Via RTM
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+### Delivery ports
+
+Escolha entre as [portas de entrega](/en/documentation/products/edge-application/#ports) suportadas para cada protocolo selecionado.
+
+1. Vá para a aba **Main Settings**.
+2. Em **HTTP Ports**, além da porta `80`, selecione a porta `8008`.
+3. Em **HTTPS Ports**, além da porta `443`, selecione a porta `8443`.
+4. Clique no botão **Save**.
+
+Aguarde alguns minutos para que as alterações se propaguem para o edge e tente acessar sua aplicação usando uma das novas portas; por exemplo, acesse `https://xxxxxxxxxx.map.azionedge.net:8443` e verifique se sua aplicação está rodando na porta personalizada.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:57]:
```suggestion
Aguarde alguns minutos para que as alterações se propaguem para o edge e tente acessar sua aplicação usando uma das novas portas. Por exemplo, acesse `https://xxxxxxxxxx.map.azionedge.net:8443` e verifique se sua aplicação está rodando na porta personalizada.
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+
+```bash
+curl -I https://xxxxxxxxxx.map.azionedge.net:8443
+```
+
+Uma resposta bem-sucedida significa que sua aplicação também está rodando na porta `8443`.
+
+### Versão e conjunto de cifras TLS
+
+Especifique a [versão mínima do TLS](/en/documentation/products/edge-application/#minimum-tls-version) e o [conjunto de cifras](/en/documentation/products/edge-application/#ciphers) suportado pela sua aplicação.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:69]: deveria ser ciphers no lugar de cifras ou estamos traduzindo esse termo?

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+
+1. Vá para a aba **Configurações Principais**.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:71]:
```suggestion
1. Vá para a aba **Main Settings**.
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+2. Em **Versão mínima do TLS**, selecione **TLS 1.2**.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:72]:
```suggestion
2. Em **Minimum TLS version**, selecione **TLS 1.2**.
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+3. Em **Lista de Cifras Suportadas**, selecione **TLSv1.2_2021**.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:73]:
```suggestion
3. Em **Supported Ciphers list**, selecione **TLSv1.2_2021**.
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 8]
```diff
@@ -0,0 +1,193 @@
...
+4. Clique no botão **Salvar**.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:74]:
```suggestion
4. Clique no botão **Save**.
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 9]
```diff
@@ -0,0 +1,193 @@
...
+
+Pode levar algum tempo para propagar suas alterações para a edge. Para verificar se suas alterações ocorreram, você pode inspecionar a página `https://xxxxxxxxxx.map.azionedge.net` usando seu navegador e localizar as configurações de segurança da aplicação. Você também pode [executar o comando DIG](/en/documentation/products/guides/run-the-dig-command/) para obter mais informações sobre suas configurações de segurança.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:76]:
```suggestion
Pode levar algum tempo para propagar suas alterações para o edge. Para verificar se suas alterações ocorreram, você pode inspecionar a página `https://xxxxxxxxxx.map.azionedge.net` usando seu navegador e localizar as configurações de segurança da aplicação. Você também pode [executar o comando DIG](/en/documentation/products/guides/run-the-dig-command/) para obter mais informações sobre suas configurações de segurança.
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 10]
```diff
@@ -0,0 +1,193 @@
...
+
+A aplicação ainda pode estar rodando em TLS 1.3, já que você selecionou a versão mínima e não a versão exata do TLS. No entanto, você pode verificar o conjunto de cifras sendo usado contra a [lista de cifras suportadas](/en/documentation/products/edge-application/#ciphers).
+
+### Módulos
+
+[Módulos](/en/documentation/products/edge-application/#modules) podem desbloquear funcionalidades e recursos adicionais para suas aplicações. Veja a [página de preços](https://www.azion.com/en/pricing/) para mais detalhes.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:82]:
```suggestion
Os [Módulos de Edge Application](/en/documentation/products/edge-application/#modules) podem desbloquear funcionalidades e recursos adicionais para suas aplicações. Veja a [página de preços](https://www.azion.com/en/pricing/) para mais detalhes.
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 11]
```diff
@@ -0,0 +1,193 @@
...
+
+Por enquanto, você ativará dois módulos. Os módulos **Application Acceleration** e **Edge Functions** serão usados quando você [ajustar suas configurações de cache](/en/documentation/products/guides/build/tune-cache-settings/), [instanciar uma função edge](/en/documentation/products/guides/build/instantiate-edge-functions/), e [trabalhar com o Rules Engine](/en/documentation/products/guides/build/work-with-rules-engine/).
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:84]:
```suggestion
Por enquanto, você ativará dois módulos. Os módulos **Application Acceleration** e **Edge Functions** serão usados quando você [ajustar suas configurações de cache](/en/documentation/products/guides/build/tune-cache-settings/), [instanciar uma função edge](/en/documentation/products/guides/build/instantiate-edge-functions/) e [trabalhar com o Rules Engine](/en/documentation/products/guides/build/work-with-rules-engine/). 
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 12]
```diff
@@ -0,0 +1,193 @@
...
+:::
+
+Em seguida, para habilitar os módulos para o sua aplicação:
+
+7. No canto superior esquerdo da página, abra o **Products menu**, representado por três linhas horizontais, e selecione **Edge Application**.
+8. Clique na edge application que você deseja configurar.
+9. Ative os módulos **Application Acceleration** e **Edge Functions**.
+10. Clique no botão **Save**.
+
+## Via API
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:106]:
```suggestion

---

## Via API
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 13]
```diff
@@ -0,0 +1,193 @@
...
+    "delivery_protocol": "http,https",
+    "http_port": [80, 8008],
+    "https_port": [443, 8443],
+    "minimum_tls_version": "tls_1_2",
+    "supported_ciphers": "TLSv1.2_2021",
+    "edge_functions": true,
+    "application_acceleration": true
+}'
+```
+
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:134]:
```suggestion
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 14]
```diff
@@ -0,0 +1,193 @@
...
+6. Teste suas alterações acessando sua aplicação usando o protocolo HTTPS e as portas personalizadas configuradas acima. Você pode acessar sua aplicação através de um navegador usando o endereço `https://xxxxxxxxxx.map.azionedge.net:8443` e inspecionar a página ou usar o seguinte comando cURL para uma resposta de status `2xx` ou `3xx`:
+
+```bash
+curl -I https://xxxxxxxxxx.map.azionedge.net:8443
+```
+
+7. Verifique o conjunto de cifras suportado inspecionando a página ou [executando o comando DIG](/en/documentation/products/guides/run-the-dig-command/).
+
+:::tip
+Confira a [documentação da API Azion](https://api.azion.com/) e a [especificação OpenAPI](https://github.com/aziontech/azionapi-openapi/) para saber mais sobre o que a API Azion pode oferecer.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:192]:
```suggestion
Confira a [documentação da API da Azion](https://api.azion.com/) e a [especificação OpenAPI](https://github.com/aziontech/azionapi-openapi/) para saber mais sobre o que a API da Azion pode oferecer.
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 15]
```diff
@@ -0,0 +1,193 @@
+---
+title: Definir configurações principais
+description: Personalize as configurações, protocolos de entrega e módulos da sua edge application.
+meta_tags: build, edge application, configuração, configurações, entrega, http, https, portas, http3
+namespace: docs_guides_build_configure_main_settings
+permalink: /documentation/products/guides/build/definir-configuracoes-principais/
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:6]:
```suggestion
permalink: /documentacao/produtos/guias/build/definir-configuracoes-principais/
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx (hunk 4 above)
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:69]:
```suggestion
Especifique a [versão mínima do TLS](/en/documentation/products/edge-application/#minimum-tls-version) e o [conjunto de cifras](/pt-br/documentacao/produtos/edge-application/#cifras) suportado pela sua aplicação.
```

**Pull request closed without merging**